{"name": "test_add[0.6-0.7-1.3]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1.2999999999999998 == 1.3", "trace": "self = <calc.test_calc.Testcalc object at 0x0000025348CB71F0>, a = 0.6, b = 0.7, expect = 1.3\n\n    @pytest.mark.parametrize('a,b,expect', get_data()[0])\n    def test_add(self, a, b, expect):\n        result = a + b\n>       assert result == expect\nE       assert 1.2999999999999998 == 1.3\n\ntest_calc.py:29: AssertionError"}, "parameters": [{"name": "a", "value": "0.6"}, {"name": "b", "value": "0.7"}, {"name": "expect", "value": "1.3"}], "start": 1603856948255, "stop": 1603856948255, "uuid": "b04a131f-ff12-40d4-a1a3-d354a1325f52", "historyId": "0c9e86906be09f1de75379098d4dca53", "testCaseId": "f8b5ecc4640e138b496eb0fb100e76eb", "fullName": "test_calc.Testcalc#test_add", "labels": [{"name": "suite", "value": "test_calc"}, {"name": "subSuite", "value": "Testcalc"}, {"name": "host", "value": "DESKTOP-NLVLGAU"}, {"name": "thread", "value": "12880-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calc"}]}